easybuild/easyconfigs/a/augur/augur-7.0.2-intel-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,130 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonBundle'
+
+name = 'augur'
+version = '7.0.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/nextstrain/augur'
+description = "Pipeline components for real-time phylodynamic analysis"
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('Biopython', '1.75', versionsuffix),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('MAFFT', '7.453', '-with-extensions'),
+    ('IQ-TREE', '1.6.12'),
+    ('VCFtools', '0.1.16'),
+    ('RAxML', '8.2.12', '-hybrid-avx2'),
+    ('FastTree', '2.1.11'),
+    ('CVXOPT', '1.1.9', versionsuffix),
```

Comment:
Ideally, this is updated to latest 1.2.4, but we'll also need to poll whether Augur is compatible with CVXOPT 1.2.x...

Reply:
I've reused your wording. Hope you don't mind :)
https://github.com/nextstrain/augur/issues/539

Reply:
:+1: